test/client_reset_test.dart

```diff
@@ -431,9 +449,9 @@ Future<void> main([List<String>? args]) async {
     await realm.syncSession.waitForUpload();
     await triggerClientReset(realm);
 
-    await waitFutureWithTimeout(manualResetFallbackCompleter.future, timeoutError: "onManualResetFallback is not reported.");
+    await manualResetFallbacKCompleter.future.wait(defaultWaitTimeout, "onManualResetFallback is not reported.");
 
-    expect(manualResetFallbackOccurred, 1);
+    expect(manualResetFallbacKOccurred, 1);
```

Comment:
```suggestion
    expect(manualResetFallbackOccurred, 1);
```

Reply:
@nielsenko I thought I have handled it all. Will fix it